src/components/Card/UserGroup/UserGroupCardList.jsx

```diff
@@ -0,0 +1,31 @@
+import asyncGetUserGroup from "../../../api/group/asyncGetUserGroup";
+import useBoundStore from "../../../store/client/useBoundStore";
+import UserGroupCard from "./UserGroupCard";
+import { useQuery } from "@tanstack/react-query";
+
+const UserGroupCardList = () => {
+  const userId = useBoundStore((state) => state.userInfo.id);
+  const isExistUserId = !!userId;
```

Comment:
```suggestion
  const isExistedUserId = !!userId;
```
변수명을 형용사로 수정해보면 좋을 것 같습니다~! `userId` 변수와 연산자를 활용하신 것 좋네요 👍 

[EXISTED | Cambridge English Dictionary](https://dictionary.cambridge.org/ko/%EC%82%AC%EC%A0%84/%EC%98%81%EC%96%B4/existed)

Reply:
```suggestion
  const hasUserId = !!userId;
```
아니면 `has`로 수정해보는 건 어떨까요~? 존재한다와 가지다 의미가 비슷한거 같아서 제안드립니다!

Reply:
한 단어로 줄어드는 효과로 `hasUserId`로 해볼게요! 그럼 존재의 유무에 대해서는 `has`로 쓰면 좋을 것 같네요!